shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/helpers/extensions/ios/UITestLogout.kt

```diff
@@ -1,15 +1,13 @@
 package co.nimblehq.blisskmmic.helpers.extensions.ios
 
-import co.nimblehq.blisskmmic.data.database.datasource.LocalDataSourceImpl
-import com.russhwolf.settings.Settings
+import co.nimblehq.blisskmmic.data.database.datasource.LocalDataSource
 import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
 
 object UITestLogout: KoinComponent {
 
     fun logOut() {
-        // TODO: Use logout use case to clear
-        val settings : Settings by inject()
-        settings.clear()
+        val localDataSource: LocalDataSource by inject()
```

Comment:
Even though we can inject the `localDataSource` directly inside the function, it is kinda a weird practice and it looks tightly-coupled here. Should we turn it into a private val instead? 🤔
```
private val localDataSource: LocalDataSource by inject()

fun logOut() {
   localDataSource.removeToken()
}
```